src/reports/ProgressCard.jsx

```diff
@@ -1,23 +1,22 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
+import Header from '../common/Header.jsx'
 
-const capitalize = str => str[0].toUpperCase() + str.slice(1)
 const ProgressCard = ({ name, id, link, title }) => {
   if (id === 'nationwide') {
     name = ''
```

Comment:
A `nationwide` id will still need to be capitalized here, though it could just be hardcoded as `id = 'Nationwide'`